src/ui/toolbar/MainToolBar.qml

```diff
@@ -356,6 +356,7 @@ Rectangle {
             anchors.top:        parent.top
             anchors.bottom:     parent.bottom
             exclusiveGroup:     mainActionGroup
+            visible:            activeVehicle ? activeVehicle.supportsMissionManager || QGroundControl.forceShowMissionManager : true
```

Comment:
Ah, true. Okay. Would you prefer that I do this entirely differently? Or just make this current version more robust?